packages/xrpl/src/models/transactions/MPTokenIssuanceCreate.ts

```diff
@@ -69,6 +78,15 @@ export interface MPTokenIssuanceCreateFlagsInterface
   tfMPTCanTrade?: boolean
   tfMPTCanTransfer?: boolean
   tfMPTCanClawback?: boolean
+
+  tmfMPTCanMutateCanLock?: boolean
```

Comment:
These flags are only available for MutableFlags field. Should it be separated into its own interface, similar to what we did with `SetFlag` options in `AccountSet`?

Reply:
1. The flags associated with Dynamic MPTs have `tmf...` in their prefix. This is used to indicate `transaction-mutable-flag`.
2. If we use a separate interface for these flags, it could potentially cause confusion. The xrpl.org documentation (and the rippled cpp code, other programming language clients) organize these flags under the `MPTokenIssuanceCreate` transaction's flags. This change would have been more impactful if it was done at the XLS level, instead of only xrpl.js deviating from the specification.